guides/common/modules/proc_creating-hosts-with-pxe-boot-provisioning.adoc

```diff
@@ -3,11 +3,22 @@
 [id="creating-hosts-with-pxe-boot-provisioning"]
 = Creating hosts with PXE boot provisioning
 
-Unattended provisioning is the simplest form of host provisioning.
+PXE boot provisioning is a method of unattended host provisioning that requires minimal interaction during the process.
+
 You enter the host details on {ProjectServer} and boot your host.
 {ProjectServer} automatically manages the PXE configuration, organizes networking services, and provides the operating system and configuration for the host.
 
-This method of provisioning hosts uses minimal interaction during the process.
+Create a host entry to add the host details.
+When you save the host entry, {Project} creates the relevant provisioning settings.
+This also includes creating the necessary directories and files for PXE booting.
+When you boot the host, the host detects the DHCP service, receives the HTTP endpoint of the installation content, and installs the operating system.
```

Comment:
I think with PXE boot the installation content is often served over TFTP, not HTTP. This specific chapter is also used in the TFTP context. Is it better to avoid the specific protocol?
```suggestion
When you boot the host, the host detects the DHCP service, receives location of the installation content, and installs the operating system.
```

It was already there, but I'd like to avoid "detects the DHCP service". Technically the host sends out a DHCP request and the DHCP service responds. Is this an idea?

```suggestion
When you boot the host, the host requests a DHCP lease.
The DHCP service responds with a lease which includes the location of the installation content.
This is used by the host to install the operating system.
```